### PR TITLE
Active and Notes tab

### PR DESCRIPTION
**Pull request by pflopez**
- UI only - adds the new layout with active and notes tab.
- Adds the interaction for adding notes (UI only/ mocks . Not saving anything)
- The new UI only shows up if the `show_share_modal` feature flag is on.

Screen Shot 2024-02-21 at 13 50 57
Screen Shot 2024-02-21 at 13 51 06

### CODE REVIEW

File: src/interface/src/app/plan/area-notes/area-notes.component.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  date: string;
+}
+
+@Component({
+  selector: 'app-area-notes',
+  templateUrl: './area-notes.component.html',
+  styleUrls: ['./area-notes.component.scss'],
+})
+export class AreaNotesComponent {
+  notes: Note[] = [
```
Comment (lastminutediorama) [inline src/interface/src/app/plan/area-notes/area-notes.component.ts:15]: is this just meant to be sample data? Could we put it in a static file or in the test spec?

Comment (pflopez) [reply src/interface/src/app/plan/area-notes/area-notes.component.ts:15]: this is just a placeholder, will go away once we plug in the actual rest call

Comment (pflopez) [reply src/interface/src/app/plan/area-notes/area-notes.component.ts:15]: I could probably just delete this, actually, as you can add notes and just see them pop im.  I'll do that.

**Review by lastminutediorama — APPROVED** [review]
lgtm, UI looks nice

**Pull request merged**